app/src/main/java/org/coepi/android/ui/home/HomeViewModel.kt

```diff
@@ -4,55 +4,92 @@ import androidx.lifecycle.LiveData
 import androidx.lifecycle.ViewModel
 import io.reactivex.Observable.just
 import io.reactivex.android.schedulers.AndroidSchedulers.mainThread
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.plusAssign
+import io.reactivex.subjects.PublishSubject
 import org.coepi.android.R.plurals
+import org.coepi.android.R.string
 import org.coepi.android.R.string.home_version
 import org.coepi.android.extensions.rx.toLiveData
 import org.coepi.android.repo.AlertsRepo
 import org.coepi.android.system.EnvInfos
 import org.coepi.android.system.Resources
 import org.coepi.android.ui.alerts.AlertsFragmentDirections.Companion.actionGlobalAlerts
 import org.coepi.android.ui.debug.DebugFragmentDirections.Companion.actionGlobalDebug
+import org.coepi.android.ui.home.HomeCardId.CHECK_IN
+import org.coepi.android.ui.home.HomeCardId.SEE_ALERTS
 import org.coepi.android.ui.navigation.NavigationCommand.ToDestination
 import org.coepi.android.ui.navigation.NavigationCommand.ToDirections
 import org.coepi.android.ui.navigation.RootNavigation
 import org.coepi.android.ui.symptoms.SymptomsFragmentDirections.Companion.actionGlobalSymptomsFragment
 
+enum class HomeCardId {
+    CHECK_IN, SEE_ALERTS
```

Comment:
These names were based off the original mock-up, I would change them to be more descriptive of what the cards are now instead.